bundles/AdminBundle/src/DependencyInjection/Compiler/ContextGuesserPass.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\AdminBundle\DependencyInjection\Compiler;
+
+use Pimcore\Http\Context\PimcoreContextGuesser;
+use Pimcore\Translation\Translator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @internal
+ */
+final class ContextGuesserPass implements CompilerPassInterface
```

Comment:
why not just use the pimcore config in AdminBundle:
```yaml
pimcore:
  context:
    admin:
      routes:
          - { path: ^/admin(/.*)?$ }
          - { route: ^pimcore_admin_ }
```

Reply:
Mhhh, i was trying to sort of "avoiding" pimcore core config reference in adminbundle yml files (not unilaterally, but in both diretion), but i agree with you, this can just be using the pimcore config